### PR TITLE
Add basic thank you (success) page.

### LINKED ISSUE #105: Redeem Success Page

### PR DESCRIPTION
**Pull request by bharathcs**
Fixes #105

the redeem page itself isn't done, but now the success page is.

https://user-images.githubusercontent.com/7360964/195887244-fceef711-fd2a-41c1-83f8-be8e14360aee.mp4

### CODE REVIEW

File: frontend/pages/redeem/[uuid]/thank-you.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import Lottie from 'lottie-react';
+import { NextPage } from 'next';
+import orangeTick from '../../../assets/orangeTickLottie.json';
+import { lottieSx, stackSx } from '../../../styles/redeem/thankYou';
+
+const ThankYou: NextPage = () => {
+  return (
+    <Stack component="main" maxWidth="md" spacing={2} sx={stackSx}>
+      <Box sx={lottieSx}>
+        <Lottie autoPlay loop={false} animationData={orangeTick} />
```
Comment (sivayogasubramanian) [inline frontend/pages/redeem/[uuid]/thank-you.tsx:11]: Change lottie to the one i sent.

File: frontend/pages/redeem/[uuid]/thank-you.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      </Box>
+      <Typography variant="h1" textAlign="center">
+        Thank you for donating!
```
Comment (sivayogasubramanian) [inline frontend/pages/redeem/[uuid]/thank-you.tsx:14]:
```suggestion
        Your donation goes a long way in making the lives of others a little better!
```

**Review by sivayogasubramanian — APPROVED** [review]
LGTM!

File: frontend/pages/redeem/[uuid]/thank-you.tsx (hunk 2 above)
Comment (bharathcs) [reply frontend/pages/redeem/[uuid]/thank-you.tsx:14]: It feels a bit overkill with the 'transform lives...' text below.

 I feel the 'Thank you for you donation' is fine given the more impactful body text.

File: frontend/pages/redeem/[uuid]/thank-you.tsx (hunk 1 above)
Comment (bharathcs) [reply frontend/pages/redeem/[uuid]/thank-you.tsx:11]: Previous:

https://user-images.githubusercontent.com/7360964/195970284-3431df07-09a6-453d-ae54-5953ce12fd79.mp4


Current:


https://user-images.githubusercontent.com/7360964/195970292-6c6ec9c9-580f-443b-9d92-d182622f979f.mp4

**Review by sivayogasubramanian — APPROVED** [review]

**Pull request merged**